doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,244 @@
+    This is added exclusively for DFU and is not expected to be used by other parts of the application.
+    Instead, use the :ref:`lib_bm_zms` and ``bm_storage`` libraries.
+
+Libraries
+=========
+
+* Added the following libraries:
+
+  * The :ref:`lib_bm_zms` library.
+  * The ``bm_storage`` library.
+  * A queue library (modified library from Zephyr).
+    This library has been added as a dependency for MCUmgr (used in DFU) offering support for threadless applications.
+
+..
+
+* ``ble_adv`` library:
+
+  * Changed the default advertising name to ``nRF_BM_device``.
+  * Corrected the :kconfig:option:`BLE_ADV_DIRECTED_ADVERTISING_HIGH_DUTY` Kconfig option.
+
+..
+
+* ``ble_conn_params library`` library:
+
+  * Updated connection parameter values to the :c:enumerator:`BLE_CONN_PARAMS_EVT_UPDATED` event.
+  * Renamed slave latency to peripheral latency according to specification.
+
+  * Fixed:
+
+    * An issue where the ATT MTU could be set to a lower value than the current value.
+    * An issue where the list of PHYs supported by the SoftDevice was not respected by the library when doing PHY updates.
+    * Data length timeout issue and the :c:func:`data_length_get` function returning wrong value.
+    * Incorrectly stated maximum ATT MTU value.
+
+Samples
+=======
+
+* General changes:
+
+  * All samples now use the logging module instead of ``printk``.
+
+  * Updated:
+
+    * The main loop sleep instructions as the :c:func:`sd_app_evt_wait` function is deprecated.
+    * Updated the main function flow to always end up in a loop with sleep instructions and deferred log processing instead of returning from the main function.
+      This ensures that log messages are printed and that the device goes into sleep mode.
+      Returning from main leaves the device in a state that prevents low power sleep and unhandled deferred log messages are not processed.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`ble_pwr_profiling_sample` sample.
+
+..
+
+* :ref:`ble_cgms_sample` sample:
+
+  * Fixed an issue with responding twice to a PHY update request.
+
+..
+
+* :ref:`ble_hrs_sample` sample:
+
+  * Removed the unused Coded PHY Kconfig fragment.
+
+..
+
+* :ref:`ble_nus_sample` sample:
+
+  * Added support for :ref:`driver_lpuarte`.
+
+..
+
+  * Fixed:
+
+    * The pin configuration to use the pins from the :file:``board_config.h`` board header.
+    * An issue in the :c:func:`ble_nus_data_send` function where passing NULL as the NUS instance could cause a segmentation fault.
+
+Peripheral samples
+------------------
+
+* Added the following samples:
+
+  * :ref:`bm_zms_sample` sample.
+  * :ref:`bm_lpuarte_sample` sample.
+  * :ref:`storage_sample` sample.
+
+DFU samples
+-----------
+
+* Added the following samples:
+
+  * :ref:`ble_mcuboot_recovery_entry_sample` sample that demonstrates how to activate DFU mode on the device using a mobile application and execute DFU over Bluetooth LE.
```

Comment:
```suggestion
  * :ref:`ble_mcuboot_recovery_entry_sample` sample that demonstrates how to activate DFU mode on the device using Bluetooth Le on a mobile application.
```

Reply:
Det focus here should be how to enter DFU mode, not that you could execute DFU (over whatever transport)